docs/Get started.md

```diff
@@ -170,7 +170,9 @@ Simply deploy to Heroku by clicking this button:
 
 ### 2. Configure Ackee
 
-Ensure that you're using the correct CORS headers by setting [`ACKEE_ALLOW_ORIGIN`](CORS%20headers.md#heroku-or-platforms-as-a-service-configuration).
+- You need to have a MongoDB instance running, for example using [mLab MongoDB](https://devcenter.heroku.com/articles/mongolab) add-on which is currently available for free. This is as simple as typing `heroku addons:create mongolab --app <YOUR_APP_NAME>` using the CLI, or using the web dashboard; more details at the [official documentation](https://devcenter.heroku.com/articles/managing-add-ons). The add-on will inject `MONGODB_URI` variable into the Ackee application so you just need to restart Heroku app _dyno_ to make Ackee process connect to MongoDB. NOTE: you need to provide your credit card details in order to use add-ons in Heroku.
```

Comment:
You should suggest a different add-on provider as mLab is being deprecated soon.

Reference: https://devcenter.heroku.com/changelog-items/1823

Reply:
@aleccool213 can you recommend any?

Reply:
Not yet 😟  I have yet to switch off of mLab.

Reply:
@aleccool213 yeah, ObjectRocket offering is too expensive for most basic needs, I wonder if it's possible/makes sense to create an Heroku add-on to run it on a free tier dyno